src/database_metrics/operations.py

```diff
@@ -0,0 +1,75 @@
+"""Operations for which metrics are captured for."""
+import requests
+from requests import Response
+
+from docker import DockerClient
+
+
+def send_query(params: dict) -> Response:
+    """
+    Send query to Epidata API running on docker MariaDB container.
+
+    Parameters
+    ----------
+    params: dict
+        Query parameters to send. List of parameters can be found here:
+        https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html#constructing-api-queries
+
+    Returns
+    -------
+    Requests Response object.
+    """
+    req = requests.get("http://localhost:10080/epidata/api.php",
+                       auth=("user", "pass"),
+                       params=params)
+    return req
+
+
+def load_data(client: DockerClient, source: str, file_pattern: str) -> bytes:
+    """
+    Ingest data into epidata database using the Python docker image.
+
+    Copies files from common_full/covidcast/receiving/`source`/`file_pattern`, which should be
+    copied into via the DockerFile, to the common/covidcast/receiving/`source`/ folder
+    for ingestion. Runs ingestion via the python command.
+
+    Parameters
+    ----------
+    client: DockerClient
+        Docker Client object containing the Python image.
+    source: str
+        Data source name
+    file_pattern: str
+        Filename of pattern to match
+
+    Returns
+    -------
+    Bytestring of Docker log, either STDOUT or STDERR
+    """
+    return client.containers.run(
+        image=client.images.get("delphi_python"),
```

Comment:
in the future, if this runs in automation, you'll likely want to build `delphi_python` prior to running it. (no action needed atm)